<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diary</class>
 <widget class="QMainWindow" name="Diary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>622</width>
     <height>433</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="selectDate">
        <property name="text">
         <string>Выберите дату: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCalendarWidget" name="calendar">
        <property name="styleSheet">
         <string notr="true">selectedDay-front-color: red;</string>
        </property>
        <property name="minimumDate">
         <date>
          <year>1900</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
        <property name="verticalHeaderFormat">
         <enum>QCalendarWidget::NoVerticalHeader</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="7" column="1">
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="hideCompleted">
        <property name="text">
         <string>Скрыть выполненные задачи</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="note">
        <property name="text">
         <string>Выбранная заметка:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTextEdit" name="taskText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 10 px solid rgb(200,244,99);
border-radius: 10px;
background-color: rgb(255,255,255)</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="todayTasks">
        <property name="text">
         <string>Показать задачи выбранного дня</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="clearTextButton">
        <property name="text">
         <string>Очистить текст</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QListWidget" name="taskList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
border: 10 px solid rgb(200,244,99);
border-radius: 10px;
background-color: rgb(255,255,255)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tasks">
        <property name="text">
         <string>Список задач:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="newNoteButton">
        <property name="text">
         <string>Создать новую заметку</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="menuButton">
        <property name="text">
         <string>В главное меню</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="clearTasksButton">
        <property name="text">
         <string>Очистить ежедневник</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="instruction"/>
    <addaction name="action_2"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="instruction">
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>hideCompleted</tabstop>
  <tabstop>todayTasks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
